app/controllers/events/view/edit/sessions-speakers.js

```diff
@@ -28,4 +28,4 @@ export default Controller.extend(EventWizardMixin, {
       );
     }
   }
-});
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204)